samcli/commands/pipeline/init/pipeline_templates_manifest.py

```diff
@@ -0,0 +1,50 @@
+"""
+Represents a manifest that lists the available SAM pipeline templates.
+Example:
+    providers:
+      - Jenkins
+      - Gitlab
+      - Github Actions
+    templates:
+      - name: jenkins-two-stages-pipeline
+        provider: Jenkins
+        location: templates/cookiecutter-jenkins-two-stages-pipeline
+      - name: gitlab-two-stages-pipeline
+        provider: Gitlab
+        location: templates/cookiecutter-gitlab-two-stages-pipeline
+      - name: Github-Actions-two-stages-pipeline
+        provider: Github Actions
+        location: templates/cookiecutter-github-actions-two-stages-pipeline
+"""
+from pathlib import Path
+from typing import Dict, List
+
+import yaml
+
+from samcli.commands.exceptions import AppPipelineTemplateManifestException
+from samcli.yamlhelper import parse_yaml_file
+
+
+class PipelineTemplateManifest:
```

Comment:
I thought a "manifest" is for the whole file? Here it suggests it's `templates` value?